app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -455,12 +248,99 @@ const validate = createValidator({
   comment: [required()],
 });
 
-const CompanyInterestPage = (props: Props) => {
-  if (props.edit && !props.companyInterest) {
+const CompanyInterestPage = () => {
+  const { companyInterestId } = useParams();
+  const edit = companyInterestId !== undefined;
```

Comment:
`edit` -> `isEditing`? Change if you want, not necessarily in scope here, but I feel that naming would make more sense

Reply:
I clearly see your point but I don't feel like refactoring variable names since I'll be going through most of the webapp codebase